integration-tests/utils/integration-helpers.js

```diff
@@ -80,6 +80,18 @@ const addUserToBilling = _.flow(withSignedInPage, withUserToken)(async ({ page,
   }, email, billingProject)
 
   console.info(`added user to: ${billingProject}`)
+
+  const userList = await page.evaluate((billingProject) => {
+    return window.Ajax().Billing.project(billingProject).listUsers()
+  }, billingProject)
+
+  function getval(email) {
```

Comment:
Functions should be camel case

Reply:
Also, we generally don't write the parentheses around a single argument.

Reply:
Fixed this